.github/workflows/ci.yml

```diff
@@ -38,10 +38,20 @@ jobs:
           distribution: 'corretto'
           cache: 'sbt'
 
-      - name: Upload to riff-raff
+      - name: Build
         run: |
-            LAST_TEAMCITY_BUILD=250
-            export GITHUB_RUN_NUMBER=$(( $GITHUB_RUN_NUMBER + $LAST_TEAMCITY_BUILD ))
             ./script/ci
+          
+      - name: Upload to riff-raff
+        uses: guardian/actions-riff-raff@v2
```

Comment:
You may need to set the [projectName](https://github.com/guardian/actions-riff-raff/#projectname) input here to ensure the name is the same in RR.